app/app.js

```diff
@@ -238,13 +246,13 @@ export default class App {
             return;
         }
 
+        this.startApp();
         this.setManagedConfig(true);
     };
 
     launchApp = async () => {
         const shouldStartCache = await this.getManagedConfig();
         if (shouldStartCache) {
-            this.startApp();
```

Comment:
if we are moving `this.startApp()` into `verifyManagedConfigCache` there's no need for storing the result of `handleManagedConfig`.

In that case we could simplify `launchApp` to only

```
launchApp = async () => {
  const shouldStart = await handleMangedConfig();
  if (shouldStart) {
      this.startApp()
   }
}
```

And then remove the need for `getManagedConfig` https://github.com/mattermost/mattermost-mobile/pull/1691/files#diff-2de9eda1655d499e934e7465e87ca0bdR71